src/sentry/utils/samples.py

```diff
@@ -206,7 +206,7 @@ def load_data(
         data["contexts"]["trace"]["span_id"] = span_id
         if trace_context is not None:
             data["contexts"]["trace"].update(trace_context)
-        if spans:
+        if spans is not None:
```

Comment:
This needs to be here so txn gen 3-0 doesn't randomly get 2 children out of nowhere

TODO: the default spans probably should attach correctly back to their parent?